packages/fether-react/src/assets/theme/globalStyle.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015-2018 Parity Technologies (UK) Ltd.
+// This file is part of Parity.
+//
+// SPDX-License-Identifier: BSD-3-Clause
+
+import { createGlobalStyle } from 'styled-components';
+
+// https://www.styled-components.com/docs/faqs
+const GlobalStyle = createGlobalStyle`
```

Comment:
Same thing here, GlobalStyle is a React components, so doesn't go inside assets/